ntbs-service/Models/NotificationBannerModel.cs

```diff
@@ -36,6 +38,10 @@ public class NotificationBannerModel
         public bool ShowPadlock;
         public string RedirectPath;
         public int? CaseManagerId;
+        public IEnumerable<string> PreviousTbServiceCodes { get; set; }
+        public IEnumerable<string> PreviousPhecCodes { get; set; }
+        public IEnumerable<string> LinkedNotificationTbServiceCodes { get; set; }
+        public IEnumerable<string> LinkedNotificationPhecCodes { get; set; }
 
         // Access level is treated as a bool for either able to edit or not. This differs from the standard PermissionLevel
         // implemented across the codebase due to there being no visual difference between no permission level and readonly

```

Comment:
This comment needs to be changed, just seen it

Reply:
I think you still need to do this too?